index.html

```diff
@@ -428,6 +437,14 @@ <h2 id="setup">Setup</h2>
     a bit difficult, so we recommend an all-in-one installer.
   </p>
 
+  <p>
+    We will teach Python using the IPython notebook, a programming environment
+    that runs in a web browser. For this to work you will need a reasonably
+    up-to-date browser. The current versions of the Chrome, Safari and
+    Firefox browsers are all supported (some older browsers, including Internet Explorer
+    version 9 and below, are not).
+  </p>
+
```

Comment:
As it says in the HTML comment a few lines above: "Remove the second paragraph if the workshop will teach Python using something other than the IPython notebook." 

Should this be moved closer to the paragraph in question? I guess it is more common to use IPython than not (and thus fewer people will need to remove this than add it if we did not include it from the template).